scripts/IA/consume_files.py

```diff
@@ -0,0 +1,101 @@
+import requests
+import os
+import argparse
+import logging
+import time
+from zipfile import ZipFile
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '-g',
+    '--guid',
+    help='This is the GUID of the target node on the OSF'
+)
+parser.add_argument(
+    '-d',
+    '--directory',
+    help='This is the target Directory for the project and its files'
+)
+parser.add_argument(
+    '-t',
+    '--token',
+    help='This is the bearer token for auth. This is required'
+)
+parser.add_argument(
+    '-u',
+    '--url',
+    help='Base URL for OSF API.'
+)
+
+BASE_URL = 'https://files.osf.io/'
+
+def consume_files(guid, token, directory, base_url=BASE_URL):
+
+    zip_url = '{}v1/resources/{}/providers/osfstorage/?zip='.format(base_url, guid)
+    path = os.path.join(directory,guid)
+    os.mkdir(path)
+    path = os.path.join(path,'files')
+    os.mkdir(path)
+    auth_header = {'Authorization': 'Bearer {}'.format(token)}
+
+    try:
+        response = requests.get(zip_url.format(guid), headers=auth_header)
+        if response.status_code == 429:
```

Comment:
I don't think waterbutler's doing this yet, because they have to get redis going, but it won't hurt to have, and I think they'll match this scheme when it gets going.